src/plugins/intel_gna/ops/util/util.hpp

```diff
@@ -139,6 +157,148 @@ inline bool is_one_dim_shapes(const ov::Shape& in_dims, const ov::Shape& out_dim
     return is_one_dim_shape(in_dims) && is_one_dim_shape(out_dims);
 }
 
+static bool is_power_activation(const ov::Node* node) noexcept {
```

Comment:
shouldn't we use name 
`IsPowerActivation` ?

Reply:
I will keep it aligned with OV code style (for example, `ngraph::op::is_output`, `ngraph::op::is_sink`, `ngraph::op::is_parameter`